apps/fz_http/test/fz_http/repo/notifier_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule FzHttp.Repo.NotifierTest do
+  use ExUnit.Case, async: false
+
+  import FzHttp.TestHelpers
+
+  alias Ecto.Adapters.SQL.Sandbox
+  alias FzHttp.Events
+  alias FzHttp.Repo
+
+  setup do
+    start_supervised!({Postgrex.Notifications, [name: Repo.Notifications] ++ Repo.config()})
+    start_supervised!(Repo.Notifier)
+    :ok = Sandbox.checkout(Repo)
+  end
+
```

Comment:
adds and removes device from wall state?

Reply:
I was getting db ownership errors when creating more than one record at a time in the test. Now that I've had some time away, I can come back and see if I can get the tests running smoothly without these errors.